pkg/services/apiserver/standalone/factory.go

```diff
@@ -87,13 +85,6 @@ func (p *DummyAPIFactory) MakeAPIServer(_ context.Context, tracer tracing.Tracer
 			tracer,
 		)
 
-	case "featuretoggle.grafana.app":
```

Comment:
no need for this one in standalone -- so just removing it rather than fixing the signature